AlphaWallet/Tokens/Coordinators/TokensCoordinator.swift

```diff
@@ -3,6 +3,7 @@
 import Foundation
 import UIKit
 import PromiseKit
+import AVFoundation
```

Comment:
Can you wrap this into a static function in `ScanQRCodeCoordinator` (or into the `CanScanQRCode` protocol's default extension, look for where `promptUserOpenSettingsToChangeCameraPermission()`  is defined, but maybe that's just for view controllers) so that `AVFoundation` doesn't need to be imported in the few classes that do now?

```
AVCaptureDevice.authorizationStatus(for: .video) != .denied
```

Reply:
what if i will expand `CanScanQRCode` with next function?
```
func requestDeviceAuthorization() -> Bool {
        guard AVCaptureDevice.authorizationStatus(for: .video) != .denied else {
            promptUserOpenSettingsToChangeCameraPermission()
            return false
        }
        return true
    }
```
and all we need to do its just 
`guard navigationController.requestDeviceAuthorization() else { return }`

Reply:
Looks good, but let's call it `ensureHasDeviceAuthorization()` instead? "request" sounds like it will request, and if the request failed, it will return false, but the request hasn't been processed yet.

Reply:
@hboon yes, shure it much better name. i will update it, and also update 
`"photos" = "Browse";`
do i need to make squash fore these commits?

Reply:
Yes, squash them since it looks like they all belong as 1 commit.